src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.cs

```diff
@@ -836,10 +836,6 @@ public static DateTime ConvertTimeToUtc(DateTime dateTime)
         /// </summary>
         internal static DateTime ConvertTimeToUtc(DateTime dateTime, TimeZoneInfoOptions flags)
         {
```

Comment:
>       { [](http://example.com/codeflow?start=1&length=8)

Could you please add assert here `dateTime.Kind != DateTimeKind.Utc`. Just in case in the future someone will call it from other places.